app/static/js/htmlmodule.js

```diff
@@ -136,7 +136,7 @@ var HTMLModule = (function(){
 	input = document.createElement('input');
 	input.setAttribute('type', 'checkbox');
 	input.setAttribute('value', 'class1');
-	var selectionId = d.name + '/' +  d.section;
+	var selectionId = d.full_name + '/' +  d.section;
```

Comment:
Is this correct field? Sorry, I didn't check. I thought it was identical to the course name.